docs/controls/access_control/ac-05.md

```diff
@@ -0,0 +1,53 @@
+# Access Control
+## AC-05 - Separation of Duties
+
+The organization:  
+a. Separates [at a minimum, data creation and control, software development and maintenance, and security functions];
+
+b. Documents separation of duties of individuals; and 
+
+c. Defines information system access authorizations to support separation of duties.
+
+### TDP Implementation
+
+**Part a.**  
+The TANF Data Portal (TDP) system has explicitly defined the roles and responsibilities that accounts perform to operate and maintain the system. This includes data creation and processing, software development, maintenance, and security implementation.
+
+Every role has flags associated with the account.  These flags are *SuperUser* and *Staff* and are set to *False* by default.  When a new TDP account is created, the OFA Admin has the ability to set the flags to *True* based on the user's job responsibilities.  Note: if an account has the *Staff* flag set to *True*, the account will have access to Django Admin, but will not have access to any of the account management functionality.
+
+The TDP system has two roles: OFA Admin and Data Prepper.
+  * OFA Admin has the following abilities: security functionality to manage TDP accounts through Django Admin, upload data on behalf of Data Preppers, upload data files locally into the web application
```

Comment:
I think I still not be following this. My understanding was that OFA Admins would be able to manage STT Users, not other OFA Admins - is that correct? If so can we specify that rather than saying `manage TDP accounts`?

Reply:
OFA Admins will have the ability to manage user accounts in the future, except for the accounts of other OFA Admins. Currently, though, The OFA Admin role grants no such ability. That ability is only given by the `is_staff` and `is_superuser` flags, which will only be the Product Owner and Tech Lead for this MVP

Reply:
> That ability is only given by the `is_staff` and `is_superuser` flags, which will only be the Product Owner and Tech Lead for this MVP

So should we remove `manage TDP accounts through Django Admin` from `OFA Admin` here?

Reply:
removed this statement from this line